packages/mdc-drag-drop/drag-collection/index.js

```diff
@@ -584,6 +634,13 @@ export class MDCDragCollection extends MDCComponent {
     this.resetState_();
   }
 
+  stopScrolling() {
+    this.activeScrollZone_ = null;
+    if (this.activeScrollTimer_) {
```

Comment:
I think we can skip this check - `cancelAnimationFrame` will silently returns if you pass it an invalid value.